### PR TITLE
Add AWS MQTT sample

### PR DESCRIPTION
**Pull request by networkfusion**

### CODE REVIEW

File: samples/MQTT/AwsMQTT/AwsMQTT/Resources.resx [hunk 1]
```diff
@@ -0,0 +1,104 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<root>
+	<!-- 
+		Microsoft ResX Schema
+
+		Version 1.3
```
Comment (josesimoes) [inline samples/MQTT/AwsMQTT/AwsMQTT/Resources.resx]: This doesn't look correct for the RESX tool nF uses.
Make sure to set the tool the nF resource parser.

File: samples/MQTT/AwsMQTT/AwsMQTT/app.config [hunk 2]
```diff
@@ -0,0 +1,15 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```
Comment (josesimoes) [inline samples/MQTT/AwsMQTT/AwsMQTT/app.config]: nF doesn't support app.config to for assembly binding. Make sure to remove this and rebuild. If VS insists on adding it there is something wrong with one of the referenced dependencies.

File: samples/MQTT/AwsMQTT/AwsMQTT/Resources.resx (hunk 1 above)
Comment (networkfusion) [reply samples/MQTT/AwsMQTT/AwsMQTT/Resources.resx]: 🤔 perhaps this is to do with the fact I created the sample when VS2019 support was not available... I will check...

Comment (networkfusion) [reply samples/MQTT/AwsMQTT/AwsMQTT/Resources.resx]: There might be a problem with the VS2019 extension adding nf resources... I am unable to do it! however the one already included seems to work fine!!!

File: samples/MQTT/AwsMQTT/AwsMQTT/app.config (hunk 2 above)
Comment (networkfusion) [reply samples/MQTT/AwsMQTT/AwsMQTT/app.config]: Done...

File: samples/MQTT/AwsMQTT/AwsMQTT/Resources.resx (hunk 1 above)
Comment (networkfusion) [reply samples/MQTT/AwsMQTT/AwsMQTT/Resources.resx]: I have updated the file manually to be correct!

**Pull request merged**